com.zettlr.Zettlr.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: com.zettlr.Zettlr
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+rename-icon: Zettlr
+command: start-zettlr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-documents
+  - --filesystem=/tmp
```

Comment:
It should probably use something along `TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"` within the launcher script instead.

Reply:
In this case i feel /tmp is a better choice because on all other distributions of Zettlr exported files will show up In /tmp. Another factor is that zettlr doesn't clean up exported files once they are created in /tmp. Flatpaks don't really clean `TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"` which would lead to it getting cluttered by random exports